common/config/src/lib.rs

```diff
@@ -344,6 +344,7 @@ impl TryFrom<RawConfig> for Config {
                 // Filter off paths that are not directories and warn about them
                 .filter_map(|d| {
                     d.clone()
+                        //TODO: Need to fix this iteration as warning now happens in dir_path.rs
```

Comment:
@c-nixon I don't think there needs to any change here, perhaps the warning might be different message?

Reply:
if we allow to specify non-existing dirs - why do we filter them here?

Reply:
This iteration just filters out anything that throws an error. Originally, if a directory didn't exist and error was thrown. Now, there is a more complex process happening down in `find_valid_path` that allows directories that don't yet exist; still propagating errors up but for different reasons. So really just the comment is no longer valid.